src/test/java/core/basesyntax/service/ParserTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.impl.ParserImpl;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParserTest {
+    private static Parser parser;
+
+    @BeforeClass
+    public static void beforeClass() {
+        parser = new ParserImpl();
+    }
+
+    @Test
+    public void parse_validData_OK() {
```

Comment:
```suggestion
    public void parse_ValidData_OK() {
```
let's check all methods and fix the names so that they follow the same pattern